### DISCUSSION TITLE
2.0.4版本，往指定表添加行策略后，表被删除，show row policy执行出错，提示表被删除，但不知道是什么表，更不知道对应行策略名称，无法删除行策略

### DISCUSSION THREAD

**Discussion by x1481962098** [apache/doris#32655] [A - General / Q&A]
![image](https://github.com/apache/doris/assets/60313214/0beafac1-86dd-4489-9702-0be0140b998b)

**Comment by yagagagaga** [comment]
可以翻一下审计日志，找出记录

**Reply by x1481962098** [reply]
明白了，谢谢，那不知道目前show row policy这个指令遇到这种情况在其它版本上也是会这样报错吗，升级版本的话会不会没有这个问题

**Reply by yagagagaga** [reply]
这个bug 已经修复了，可以等下一个版本发布 https://github.com/apache/doris/pull/32310
